MojioSDK/mojiosdksrc/src/main/java/io/moj/mobile/android/sdk/MojioClient.java

```diff
@@ -291,18 +250,13 @@ public void onFailure(ResponseError error) {
         }
 
         String entityPath = String.format("Login/%s?secretKey=%s&userOrEmail=%s&password=%s&minutes=%d",
```

Comment:
Maybe we should not proceed if the encoding is invalid as the username and/or password will be null.